docs/spec/spec.md

```diff
@@ -944,25 +946,49 @@ response when returning `anydata` directly from a resource method.
 | HEAD              | Retrieve headers                                              | 200 OK                  |
 | OPTIONS           | Retrieve permitted communication options                      | 200 OK                  |
 
-#### 2.3.6. Introspection resource
+#### 2.3.6. OAS resources
```

Comment:
Can we use the full version in the title: 'OpenAPI specification resources'?

Reply:
Fixed with [1f5b0b7d0d79e46f4c85d64f44758c8039d890eb](https://github.com/ballerina-platform/module-ballerina-http/pull/2039/commits/1f5b0b7d0d79e46f4c85d64f44758c8039d890eb)